### PR TITLE
docs: Add the changelog of 3.18.2

### PR DESCRIPTION
**Pull request by dengfuping**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-3.18.2/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-3.18.2/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,23 @@ timeline: true
 
 ---
 
+## 3.18.2
+
+`2019-05-20`
+
+- 🐞 修复 Button 组件中插入文本时，空格丢失的问题。[#15342](https://github.com/ant-design/ant-design/issues/15342)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 这个，是多余的。

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,23 @@ timeline: true
...
+- 🐞 修复 Carousel 组件的 `children` 数量变化时，面板指示点的选中状态不正常的问题。[#16583](https://github.com/ant-design/ant-design/issues/16583)
+- 🐞 升级 `rc-calendar` 依赖到 `9.13.3` 版本，以修复 DatePicker 组件在失去焦点时面板不会关闭的问题。[#16588](https://github.com/ant-design/ant-design/issues/16588)
+- 🐞 修复 Form.Item 组件带有 help 信息，并且 margin-bottom 为负数时，导致之后的元素样式错乱的问题。[#16584](https://github.com/ant-design/ant-design/pull/16584) [@sbusch](https://github.com/sbusch)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: help 和 margin-bottom 加 `

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (dengfuping) [reply CHANGELOG.zh-CN.md]: - done ~

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (dengfuping) [reply CHANGELOG.zh-CN.md]: - done ~

**Review by afc163 — APPROVED** [review]

**Pull request merged**